packages/ignite-basic-structure/templates/package.json.unholy.ejs

```diff
@@ -23,46 +23,46 @@
     "flow": "flow --show-all-errors"
   },
   "dependencies": {
-    "apisauce": "^0.6.0",
+    "apisauce": "^0.8.0",
     "format-json": "^1.0.3",
     "lodash": "^4.17.2",
     "querystringify": "0.0.4",
-    "ramda": "^0.22.1",
+    "ramda": "^0.23.0",
     "react": "~15.4.0-rc.4",
-    "react-native": "0.38.0",
-    "react-native-animatable": "^0.6.0",
-    "react-native-config": "^0.1.2",
+    "react-native": "0.40.0",
+    "react-native-animatable": "^1.1.0",
+    "react-native-config": "^0.2.1",
     "react-native-device-info": "^0.9.6",
     "react-native-drawer": "^2.3.0",
-    "react-native-i18n": "^0.1.1",
-    "react-native-maps": "^0.11.0",
+    "react-native-i18n": "^1.0.0",
+    "react-native-maps": "0.13.0",
     "react-native-router-flux": "^3.37.0",
-    "react-native-vector-icons": "^3.0.0",
-    "react-redux": "^4.4.6",
+    "react-native-vector-icons": "^4.0.0",
+    "react-redux": "^5.0.2",
     "redux": "^3.6.0",
     "redux-logger": "^2.7.4",
-    "redux-persist": "^3.5.0",
-    "redux-saga": "^0.13.0",
-    "reduxsauce": "0.2.0",
-    "seamless-immutable": "^6.3.0"
+    "redux-persist": "^4.1.0",
+    "redux-saga": "^0.14.3",
+    "reduxsauce": "0.4.1",
+    "seamless-immutable": "^7.0.1"
```

Comment:
I love that you updated all of these.  I know some were necessary, but it's always a key update.